consensus/src/epoch_manager.rs

```diff
@@ -941,6 +949,7 @@ impl<P: OnChainConfigProvider> EpochManager<P> {
             verifier: (&validator_set).into(),
         };
 
+        let features: anyhow::Result<Features> = payload.get();
```

Comment:
nit: we should keep it consistent with other two, check if it's error and log then unwrap with default.

this also makes me feel better if it's on chain consensus config, since we already pass the config to round manager

Reply:
fixed